python/ray/rllib/a3c/a3c.py

```diff
@@ -73,8 +73,13 @@ def get_completed_rollout_metrics(self):
         return completed
 
     def start(self):
+        logdir = self.logdir
+        if logdir.startswith("s3"):
+            print("WARNING: TensorFlow logging to S3 not supported by TensorFlow,"
+                  " logging to /tmp/ instead")
+            logdir = "/tmp/"
         summary_writer = tf.summary.FileWriter(
-            os.path.join(self.logdir, "agent_%d" % self.id))
+            os.path.join(logdir, "agent_%d" % self.id))
```

Comment:
[1] In `common.py`, the tmp directory is a `/tmp/ray/ + prefix`. It makes sense to make that consistent and also to abstract all the logging file naming code into a separate file, perhaps in a separate PR.